src/field.rs

```diff
@@ -749,6 +790,46 @@ mod tests {
         let serialized_vec = F::slice_into_byte_vec(&test_inputs);
         let deserialized = F::byte_slice_into_vec(&serialized_vec).unwrap();
         assert_eq!(deserialized, test_inputs);
+
```

Comment:
it would be good to add a test that tries to create an element from a u128 value not in range [0,p).